generators/app/index.js

```diff
@@ -20,6 +20,12 @@ class NgxGenerator extends Generator {
       required: false
     });
 
+    this.option('external-chrome', {
```

Comment:
Could you move this to `options.json`? This is where all options are declared :)